sbt-dumpschema-plugin/src/test/scala/org/virtuslab/akkasaferserializer/PluginSuit.scala

```diff
@@ -0,0 +1,29 @@
+package org.virtuslab.akkasaferserializer
+
+import better.files.File
+import org.scalatest.matchers.should
+import org.scalatest.wordspec.AnyWordSpecLike
+import org.virtuslab.akkasaferserializer.compiler.DumpCompiler
+
+import java.io.{BufferedReader, InputStreamReader}
+import java.nio.charset.StandardCharsets
+import java.util.stream.Collectors
+
+class PluginSuit extends AnyWordSpecLike with should.Matchers {
+  private def getResourceAsString(name: String) =
+    new BufferedReader(new InputStreamReader(getClass.getClassLoader.getResourceAsStream(name), StandardCharsets.UTF_8))
+      .lines()
+      .collect(Collectors.joining("\n"))
+
+  val code = List(getResourceAsString("MySerializable.scala"), getResourceAsString("Data.scala"))
+
+  "DumpCompilerPlugin" should {
+    "dump main constructor of relevant classes" in {
+      File.usingTemporaryDirectory() { directory =>
```

Comment:
Whoops I'm getting a weird error locally here 🤔 
```
[info] PluginSuit:
[info] DumpCompilerPlugin
[info] - should dump main constructor of relevant classes *** FAILED ***
[info]   java.io.UncheckedIOException: java.nio.file.AccessDeniedException: /tmp/systemd-private-e345934280164c05bfa37d23a62f3681-geoclue.service-rlGgEh
[info]   at java.base/java.nio.file.FileTreeIterator.fetchNextIfNeeded(FileTreeIterator.java:87)
[info]   at java.base/java.nio.file.FileTreeIterator.hasNext(FileTreeIterator.java:103)
[info]   at java.base/java.util.Spliterators$IteratorSpliterator.tryAdvance(Spliterators.java:1811)
[info]   at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.lambda$initPartialTraversalState$0(StreamSpliterators.java:294)
[info]   at java.base/java.util.stream.StreamSpliterators$AbstractWrappingSpliterator.fillBuffer(StreamSpliterators.java:206)
[info]   at java.base/java.util.stream.StreamSpliterators$AbstractWrappingSpliterator.doAdvance(StreamSpliterators.java:161)
[info]   at java.base/java.util.stream.StreamSpliterators$WrappingSpliterator.tryAdvance(StreamSpliterators.java:300)
[info]   at java.base/java.util.Spliterators$1Adapter.hasNext(Spliterators.java:681)
[info]   at scala.collection.convert.Wrappers$JIteratorWrapper.hasNext(Wrappers.scala:43)
[info]   at better.files.Dispose$FlatMap$Implicits$traversableFlatMap$.$anonfun$apply$3(Dispose.scala:168)
[info]   ...
[info]   Cause: java.nio.file.AccessDeniedException: /tmp/systemd-private-e345934280164c05bfa37d23a62f3681-geoclue.service-rlGgEh
[info]   at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:90)
[info]   at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
[info]   at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
[info]   at java.base/sun.nio.fs.UnixFileSystemProvider.newDirectoryStream(UnixFileSystemProvider.java:432)
[info]   at java.base/java.nio.file.Files.newDirectoryStream(Files.java:471)
[info]   at java.base/java.nio.file.FileTreeWalker.visit(FileTreeWalker.java:300)
[info]   at java.base/java.nio.file.FileTreeWalker.next(FileTreeWalker.java:373)
[info]   at java.base/java.nio.file.FileTreeIterator.fetchNextIfNeeded(FileTreeIterator.java:94)
[info]   at java.base/java.nio.file.FileTreeIterator.hasNext(FileTreeIterator.java:103)
[info]   at java.base/java.util.Spliterators$IteratorSpliterator.tryAdvance(Spliterators.java:1811)
[info]   ...
[info] Run completed in 8 seconds, 426 milliseconds.
[info] Total number of tests run: 3
[info] Suites: completed 2, aborted 0
[info] Tests: succeeded 2, failed 1, canceled 0, ignored 0, pending 0
[info] *** 1 TEST FAILED ***
[error] Failed tests:
[error] 	org.virtuslab.akkasaferserializer.PluginSuit
```

Reply:
Any update on the error? Does it still occur?

Reply:
Not happening anymore 😌